test/qa-tests/jstests/libs/test_background_ops.js

```diff
@@ -173,15 +181,14 @@ startParallelOps = function(mongo, proc, args, context) {
       setResult = function(result, err) {
         return context.setResult(db.getMongo(), context.procName, result, err);
       };
-    },
+    }
   };
 
   var bootstrapper = function(stored) {
     var procContext = stored.procContext;
     procContext.setup(procContext, stored);
 
     var contexts = stored.contexts;
-    eval("contexts = " + contexts); // eslint-disable-line no-eval
```

Comment:
The linter complained about `eval` since it's an unsafe javascript function -- apparently it should never really be used. Seems like we don't need to use it anyway.